client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,214 @@
+									label={ translate( 'Add a page' ) }
+									iconSrc="/calypso/images/customer-home/page.svg"
+								/>
+								<ActionBox
+									action={ () => page( `/post/${ siteSlug }` ) }
+									label={ translate( 'Write blog post' ) }
+									iconSrc="/calypso/images/customer-home/post.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Customize theme' ) }
+									iconSrc="/calypso/images/customer-home/customize.svg"
+								/>
+								<ActionBox
+									action={ () => page( `/themes/${ siteSlug }` ) }
+									label={ translate( 'Change theme' ) }
+									iconSrc="/calypso/images/customer-home/theme.svg"
+								/>
+								<ActionBox
+									action={ customizeUrl }
+									label={ translate( 'Edit menus' ) }
+									iconSrc="/calypso/images/customer-home/menus.svg"
+								/>
+								<ActionBox
+									action="https://en.support.wordpress.com/images/"
+									label={ translate( 'Change images' ) }
+									iconSrc="/calypso/images/customer-home/images.svg"
+								/>
+								<ActionBox
+									action="https://logojoy.grsm.io/looka"
+									label={ translate( 'Design a logo' ) }
+									iconSrc="/calypso/images/customer-home/logo.svg"
+								/>
+								<ActionBox
+									action="https://wordpress.com/email/domain.wordpress.com"
+									label={ translate( 'Add G Suite' ) }
+									iconSrc="/calypso/images/customer-home/gsuite.svg"
+								/>
+							</div>
+						</Card>
+					</div>
+					<div className="customer-home__layout-col">
+						<Card>
+							<CardHeading>{ translate( 'Grow & Earn' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Grow my audience and earn money' ) }
+							</h6>
+							<VerticalNav className="customer-home__card-links">
+								<VerticalNavItem path={ `/marketing/connections/${ siteSlug }` }>
+									{ translate( 'Share my site' ) }
+								</VerticalNavItem>
+								<VerticalNavItem path={ `/marketing/tools/${ siteSlug }` }>
+									{ translate( 'Grow my audience' ) }
+								</VerticalNavItem>
+								<VerticalNavItem path={ `/earn/${ siteSlug }` }>
+									{ translate( 'Earn money' ) }
+								</VerticalNavItem>
+							</VerticalNav>
+						</Card>
+						<Card>
+							<CardHeading>{ translate( 'Support' ) }</CardHeading>
+							<h6 className="customer-home__card-subheader">
+								{ translate( 'Get all of the help you need' ) }
+							</h6>
+							<div className="customer-home__card-support">
+								<img
+									src="/calypso/images/customer-home/happiness.png"
+									alt={ translate( 'Support' ) }
+								/>
+								<VerticalNav className="customer-home__card-links">
+									<VerticalNavItem path="https://en.support.wordpress.com/" external>
+										{ translate( 'Support docs' ) }
+									</VerticalNavItem>
+									<VerticalNavItem path="https://wordpress.com/help/contact" external>
```

Comment:
I think that this would be helpful to keep the flows when not on WordPress.com

```suggestion
									<VerticalNavItem path="/help/contact" external>
```